templates/default/config/siteConfig.js

```diff
@@ -29,6 +31,8 @@ const defaultConfig = {
 export const siteConfig = {
   ...defaultConfig,
   ...config,
+  contentExclude: [...defaultConfig.contentExclude, ...config.contentExclude],
```

Comment:
It's not necessary. The behavior we want to achieve is the ability to set/override contentExclude or contentInclude by the user.